rdg_datasets/unique_tags_ai/generate.py

```diff
@@ -0,0 +1,33 @@
+import os
+import pathlib
+import subprocess
+
+import csv_datasets
+from libuprev import constants, rdg_generate, rdg_import
+from libuprev.uprev_config import Config
+
+local_path = pathlib.Path(os.path.dirname(os.path.realpath(__file__)))
+csv_dataset = "unique_tags_ai"
+csv_path = csv_datasets.csv_dataset_dir / csv_dataset
+import_args = ["--ids-are-integers"]
+node_file = "node_schema.txt"
+edge_file = "edge_schema.txt"
+
+
+def uprev(config: Config, new_storage_format_version: int) -> pathlib.Path:
```

Comment:
Please rename this file "import.py" per https://github.com/KatanaGraph/test-datasets#the-following-scripts-are-conditionally-present-for-each-rdg-one-of-the-following-options-must-be-present

Reply:
done